apps/financial-remedy/finrem-cos/preview.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: finrem-cos
+  namespace: financial-remedy
+spec:
+  releaseName: finrem-cos
+  values:
+    java:
+      image: hmctspublic.azurecr.io/finrem/cos:preview-XXXX-XXXX #{"$imagepolicy": "flux-system:preview-finrem-cos"}
```

Comment:
preview.yaml is not needed, you don't add to preview via Flux - its done via your Jenkins pipeline

Reply:
@damongreen123 same applies for preview-repo and image-policy file too :)

Reply:
yep I was copying idam. realise its different, I did wonder.